src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -613,6 +613,7 @@ private static void assertLogMatches(WorkflowRun build, String regexp) throws IO
                 Queue.Item[] items = Queue.getInstance().getItems();
                 assertEquals(1, items.length);
                 assertEquals(p, items[0].task.getOwnerTask());
+                assertEquals(b, items[0].task.getOwnerExecutable());
```

Comment:
Test is covering something unrelated, but seemed as good a place as any to cover the new method.